src/mmw/apps/modeling/views.py

```diff
@@ -231,6 +262,34 @@ def scenario_custom_weather_data(request, scen_id):
 
         elif request.method == 'DELETE':
             if scenario.weather_custom.name:
+                # Check if any other scenarios use the same weather file
+                others = (scenario.project.scenarios
+                          .exclude(id=scenario.id)
+                          .filter(weather_type=WeatherType.CUSTOM,
+                                  weather_custom=scenario.weather_custom)
+                          .values_list('name', flat=True))
+
+                if others.count() > 0:
+                    errors.append('Cannot delete weather file.'
+                                  ' It is also used in: "{}".'
+                                  ' Either delete those scenarios, or set them'
+                                  ' to use available weather before deleting'
```

Comment:
Consider changing this to "to use Available Data" or "to use another available weather data set".  I think it may be helpful to use consistent  label terminology.

Reply:
Fixed in 6ceff3b